src/explorer.ts

```diff
@@ -200,6 +200,7 @@ class KubernetesWorkloadFolder extends KubernetesFolder {
             new KubernetesResourceFolder(kuberesources.allKinds.job),
             new KubernetesResourceFolder(kuberesources.allKinds.cronjob),
             new KubernetesResourceFolder(kuberesources.allKinds.pod),
+            new KubernetesCRDFolder(kuberesources.allKinds.crd),
```

Comment:
I'm doubtful about putting this under Workloads.  A CRD may represent a workload, but it may also represent a piece of configuration data that does not directly translate into executable work, such as a storage resource or an external service (a la svccat scenarios).

If other k8s UIs put CRDs under Workloads then let's go along with that, otherwise I'd suggest putting this at the top level.

Reply:
You've got a point, changing that as well.

Reply:
Agree. CRDs could literally be anything, so top-level for now would be my take, but Ivan decides.